multi-agent/tests/unit/sharing/test_share_cloner_builtin_handling.py

```
@@ -0,0 +1,106 @@
+"""Unit tests: ShareCloner._compute_closure() built-in resource handling.
+
+Verifies that:
+- PUBLIC built-ins are skipped (not cloned) and don't raise.
+- DRAFT built-ins trigger ShareCloneError.
+- Mixed closures (custom depends on public built-in) succeed and only clone custom.
+"""
+from typing import Any, Optional, Tuple
```

Ruff (0.16.0)

[warning] `typing.Tuple` is deprecated, use `tuple` instead

(UP035)